src/uShip.Logging/LogBuilders/FluentLogDataBuilder.cs

```diff
@@ -21,7 +23,7 @@ private class FluentLogDataBuilder : IFluentLoggerWriter
             private Exception _exception;
             private string _sql;
             private IEnumerable<KeyValuePair<string, object>> _sqlParameters;
-            private readonly IDictionary<string, object> _data = new Dictionary<string, object>();
+            internal readonly IDictionary<string, object> _data = new Dictionary<string, object>();
```

Comment:
I'd prefer you expose a property (maybe internal) that returns an `IReadOnlyDictionary<string, object>` which you label as "for unit testing purposes only".  Or not.  That would ultimately be safe to expose without caveat, whereas this isn't.

https://msdn.microsoft.com/en-us/library/hh136548(v=vs.110).aspx